test/integration/suite_test.go

```diff
@@ -39,9 +44,14 @@ func TestMain(m *testing.M) {
 
 	fmt.Println("INFO: setting up test environment")
 	kongbuilder := kong.NewBuilder()
-	if dbmode == "postgres" {
-		kongbuilder = kongbuilder.WithPostgreSQL()
+	if enterprise == "yes" {
```

Comment:
I might just be missing something: where is this variable declared? is it not in the test vars I might be missing something :thinking:

Reply:
changed to https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-b5e061fd98698fd6cfe27cc0e162263d736df3560a8b8d58e80bab161887c7b8R43